test/Microsoft.TemplateEngine.Core.UnitTests/SetFlagTests.cs

```diff
@@ -147,5 +148,41 @@ public void ValidateDontEmitFlagDoesntAddNewline()
             IProcessor processor = Processor.Create(engineConfig, operations);
             RunAndVerify(originalValue, expectedValue, processor, 9999);
         }
+
+        // Using //-:cnd:noEmit on the first line of *.cs file corrupts file content when templating new project
+        [Fact(DisplayName = nameof(Bug2913))]
+        public void Bug2913()
```

Comment:
should we use better name? :) e.g. `ValidateConditionOnStartOfFile'`

Reply:
I will rename